src/de/duenndns/ssl/MemorizingTrustManager.java

```diff
@@ -495,11 +495,9 @@ private static String certHash(final X509Certificate cert, String digest) {
 		}
 	}
 
-	private static void certDetails(StringBuilder si, X509Certificate c) {
+	private void certDetails(StringBuilder si, X509Certificate c) {
 		SimpleDateFormat validityDateFormater = new SimpleDateFormat("yyyy-MM-dd");
 		si.append("\n");
-		si.append(c.getSubjectDN().toString());
```

Comment:
Is there a reason to remove the DN? I think that posting that does make sense, and maybe we should just put the SANs right below it.

Having the SANs at the end of the cert info doesn't make sense for sure.

Reply:
Yes, I think that it shouldn't be separated from the domain list. All domains should be in one place.

Reply:
Yes, one list at the top of the certificate info.